.circleci/config.yml

```diff
@@ -10,6 +10,59 @@ version: 2
 # during the actual CI testing is no longer necessary.
 
 jobs:
+  "CheckFormat":
+    docker:
+      - image: ubuntu:bionic
+    steps:
+      - checkout
+      - run: |
+          apt-get -q update -y
+          apt-get -q install -y apt-transport-https ca-certificates gnupg software-properties-common wget
```

Comment:
Aside from wget and gnupg, why do we need these other packages?

Reply:
`software-properties-common` adds apt-add-repository and the others allow verifying https repos. This is just the general incantation for taking a minimal ubuntu image to a slightly more useful one. We could filter it down on a repo/ppa basis for this particular use, but since these images are for dev environments instead of being deployed to production, it seems better to err on the side of ease of use rather than image size or excess dependencies.

Reply:
Got it. Thanks for the clarification.

Reply:
Another bit of context now that I remember :D : 
I'm trying to reduce the configuration drift between travis and circleci. Travis uses a xenial-based VM and so these additional packages in CircleCI bring its minimal docker xenial image more inline with the Travis Xenial VM. Folks, now, should be able to "transcribe" travis sources and packages to their corresponding CircleCI equivalents.

From my brief experience, the UX for CircleCI seems nicer than Travis (ssh to debug FTW), and that's why I put the new CI configurations in CircleCI.